webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -90,7 +90,7 @@ const Disk = ({
         minValue={1}
         format={v => `${v} GB`}
         className="text-vmware-size"
-        onChange={newValues => updateDisk('sizeGb', newValues)}
+        onValueChange={value => updateDisk('sizeGb', { target: { value }})}
```

Comment:
little hack, to make the event processer work, I would like to hear if we would like to have `onValueChange` on every input, or not :)